notes-service/src/main/java/org/exoplatform/wiki/service/impl/NoteServiceImpl.java

```diff
@@ -1196,9 +1196,11 @@ public void importNote(Page note, Page parent, Wiki wiki, String conflict, Ident
 
           }
           if (conflict.equals("duplicate")) {
-            int i = 1;
+            String title = note.getTitle();
+            int i = title.lastIndexOf("_") != -1 ? Integer.valueOf(title.substring(title.lastIndexOf("_") + 1)) + 1 : 1;
```

Comment:
I think that we may have a NumberFormatException here since the substring after the last "_" can be not numeric. should be sure that this substring can be converted to int before incrementing it.